src/com/company/Product.java

```diff
@@ -71,6 +73,24 @@ public LocalDateTime getDeliveryDueDate() throws Exception {
         }
         throw new Exception("Category could not find");
     }
+
+    public static void listAllProducts() {
```

Comment:
I could not see where this method was called...
Lets use it here in the main instead of this code block

for (Product product : StaticConstants.PRODUCT_LIST) {
              DateTimeFormatter df = DateTimeFormatter.ofPattern("dd.MM.yyyy");
              try {
                System.out.println();
                System.out.println(
                    "Product name: \"" + product.getName() +
                            "\" - ID: " + product.getId() +
                            " - Price: $" + product.getPrice() +
                        " - Category: " + product.getCategoryName() +
                        " - Stock: " + product.getRemainingStock() +
                        " - Delivery Due Date: " + product.getDeliveryDueDate().format(df));
              } catch (Exception e) {
                System.out.println(e.getMessage());
              }
            }